app/models/kbo-administrative-unit.js

```diff
@@ -0,0 +1,12 @@
+import { belongsTo, attr } from '@ember-data/model';
+import OrganizationModel from './organization';
+
+export default class KboAdministrativeUnitModel extends OrganizationModel {
+  @attr rechtsvorm;
+  @attr shortName;
+  @attr startDate;
+  @belongsTo('administrative-unit', {
+    inverse: 'kboAdministrativeUnit',
+  })
```

Comment:
**Issue:** Since the ember update, you need to add extra parameters in the relationship decorator. 

```suggestion
  @belongsTo('administrative-unit', {
    inverse: 'kboAdministrativeUnit',
    async: true,
  })
```